src/Automation/Automation/Common/AutomationPSClient.cs

```diff
@@ -2013,7 +2013,7 @@ private DirectoryInfo WriteRunbookToFile(string outputFolder, string runbookName
 
             var fileExtension = IsGraphRunbook(runbookType) ? Constants.SupportedFileExtensions.Graph : Constants.SupportedFileExtensions.PowerShellScript;
 
-            var outputFilePath = outputFolderFullPath + "\\" + runbookName + fileExtension;
+            var outputFilePath = Path.Join(outputFolderFullPath, runbookName + fileExtension);
```

Comment:
Did you use system.io namespace in this file?

Reply:
I did use System.IO namespace in this file but seems like Path.Join is only available in .NET Core. It's not in .NET Framework, and thus not available through netstandard. Causing the Error :"Path" does not contain a defination for "Join". A solution could be adding Path.DirectorySeparatorChar directly or using Path.Combine